components/marketing_signup_modal/index.coffee

```diff
@@ -0,0 +1,52 @@
+Backbone = require 'backbone'
+url = require 'url'
+sd = require('sharify').data
+FlashMessage = require '../flash/index.coffee'
+modalize = require '../modalize/index.coffee'
+template = -> require('./index.jade') arguments...
```

Comment:
:-P `arguments` are a keyword in Javascript. This line is basically wrapping the template in an anonymous function to defer it's parsing to runtime.

This is an awkwardly useful way to unit test these browserified files in Node. Admittedly there's probably a better way to do this, maybe running the code through Browserify before requiring it in tests.